foundry/foundry.py

```diff
@@ -1,4 +1,6 @@
 
+import enum
+from sklearn import datasets
```

Comment:
Is this used?

Reply:
No, I don't think so. I don't recall adding them either. I've removed them now.